roles/ceph-rgw-loadbalancer/defaults/main.yml

```diff
@@ -12,6 +12,10 @@ haproxy_frontend_port: 80
 haproxy_frontend_ssl_port: 443
 haproxy_frontend_ssl_termination: False
 
+haproxy_stats_socket_enabled: false
+haproxy_stats_socket_group: haproxy
+haproxy_stats_socket_mode: '0660'
```

Comment:
Can we instead make the changes that you plan on submitting upstream? e.g.

```
haproxy_stats_socket_group: root
haproxy_stats_socket_mode: 0755
```

Then we can override these as we need in our own `group_vars/` definitions.

Reply:
The benefit of doing so is that we won't need to change any `group_vars` once these changes are merged upstream - makes life simpler.

Reply:
Done.